pages/1.11/installing/upgrading/index.md

```diff
@@ -78,46 +79,56 @@ These steps must be performed for version upgrades and cluster configuration cha
 - Ensure that Marathon event subscribers are disabled before beginning the upgrade. Leave them disabled after completing the upgrade, as this feature is now deprecated.
-**Note: Marathon event subscribers are disabled by default, please check if the line `--event_subscriber "http_callback"` has been added to `sudo vi /opt/mesosphere/bin/marathon.sh` on your master node(s). If this is the case you will need to remove that line in order to disable event subscribers.**
+
+**Note:** Marathon event subscribers are disabled by default, check if the line `--event_subscriber "http_callback"` has been added to `sudo vi /opt/mesosphere/bin/marathon.sh` on your master node(s). In this case, you will need to remove that line in order to disable event subscribers.** [enterprise type="inline" size="small" /]
+
 - Verify that all Marathon application constraints are valid before beginning the upgrade. Use [this script](https://github.com/mesosphere/public-support-tools/blob/master/check-constraints.py) to check if your constraints are valid.
-- [Back up your cluster](/1.11/administering-clusters/backup-and-restore/).
+- [Back up your cluster](/1.11/administering-clusters/backup-and-restore/). [enterprise type="inline" size="small" /]
 - Optional: You can add custom [node and cluster healthchecks](/1.11/installing/ent/custom/node-cluster-health-check/#custom-health-checks) to your `config.yaml`.
+- Verify that all your masters are in a healthy state: 
+   - Check the Exhibitor UI to confirm that all masters have joined the quorum successfully (the status indicator will show green). The Exhibitor UI is available at `http://<dcos_master>:8181/`.
+   - Verify that `curl http://<dcos_master_private_ip>:5050/metrics/snapshot` has the metric `registrar/log/recovered` with a value of `1` for each master.
+
 
 ## Bootstrap Node
 
+[enterprise]
+### Enterprise users 
+[/enterprise]
+
 Choose your desired security mode and then follow the applicable upgrade instructions.
 
 - [Installing DC/OS 1.11 without changing security mode](#current-security)
 - [Installing DC/OS 1.11 in permissive mode](#permissive)
 - [Installing DC/OS 1.11 in strict mode](#strict)
 
-# <a name="current-security"></a>Installing DC/OS 1.11 without changing security mode
+#### Installing DC/OS 1.11 without changing security mode 
 This procedure upgrades a DC/OS 1.10 cluster to DC/OS 1.11 without changing the cluster's [security mode](/1.11/installing/ent/custom/configuration/configuration-parameters/#security-enterprise).
 
 1.  Copy your existing `config.yaml` and `ip-detect` files to an empty `genconf` folder on your bootstrap node. The folder should be in the same directory as the installer.
-1.  Merge the old `config.yaml` into the new `config.yaml` format. In most cases the differences will be minimal.
+2.  Merge the old `config.yaml` into the new `config.yaml` format. In most cases the differences will be minimal.
```

Comment:
Markdown supports auto numbering and doesn't require the user to explicitly make a change to these numbers. Would it be possible that we can revert this? https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list

Reply:
@bernadinm I've showed you an example on how the numbering is impacted between sections. We'll figure out another way to fix this. Ty for this share.